docs/features/sinks/azure-application-insights.md

```diff
@@ -57,4 +57,66 @@ if(string.IsNullOrEmpty(key) == false)
 ILogger logger = loggerConfig.CreateLogger();
 ```
 
+### Q: Where can I initialize the logger in an ASP.NET Core application or other hosted service?
+
+The Application Insights instrumentation key is typically available via the configuration or via an `ISecreteProvider`.  It's best if the Logger is created and assigned to the Serilog `Log.Logger` as soon as possible.  The easiest place to do that in an ASP.NET application  is in the `ConfigureServices` method of the `Program` class.
+
+Another place where the logger can be initialized is in the `ConfigureAppConfiguration` method of the `IHostBuilder`:
+
+```csharp
+var host = Host.CreateDefaultBuilder()
+               .ConfigureAppConfiguration((context, configBuilder) =>
+               {
+                   var config = configBuilder.Build();
+
+                   var instrumentationKey = config["ApplicationInsights:InstrumentationKey"];
+
+                   var logConfiguration = new LoggerConfiguration()
+                                           .Enrich.FromLogContext()
+                                           .WriteTo.Console();
+
+                    var appInsightsInstrumentationKey = builtConfig["ApplicationInsights:InstrumentationKey"];
+
+                    if (!String.IsNullOrWhiteSpace(appInsightsInstrumentationKey))
+                    {
+                        logConfiguration.WriteTo.AzureApplicationInsights(appInsightsInstrumentationKey);
+                    }
+
+                    Log.Logger = logConfiguration.CreateLogger();
+               })
+               .UseSerilog()
+               .Build();
+```
+
```

Comment:
Maybe a link to the Arcus Security library is useful here.

Reply:
Added a sample where Arcus Security is used + added a link to the Arcus.Security GH project